app/src/main/java/com/example/cats_or_dogs2/CatAdapter.java

```diff
@@ -14,10 +14,10 @@
 
 public class CatAdapter extends RecyclerView.Adapter<CatAdapter.viewHolder> {
     Context context;
-    ArrayList<CatManager> allCats;
+    ArrayList<CatModel> allCats;
```

Comment:
I'm guessing you want your `CatAdapter` class to be responsible for displaying a list of Cats, so this should be a list of `CatModel` instances rather than `CatManager`.